io/disk/Avago_storage_adapter/avago9361_vd.py

```diff
@@ -0,0 +1,196 @@
+                          'r00': [4, None, 'Even'], 'r10': [4, None, 'Even'],
+                          'r50': [6, None, 'Multiple3'],
+                          'r60': [6, None, 'Multiple3']}
+        self.value = self.dict_raid[self.raid_level]
+
+        if self.number_of_disk < self.value[0]:
+            self.skip("Please give enough number of drives to create %s"
+                      % self.raid_level)
+        if self.value[2] == 'Multiple3':
+            if self.number_of_disk % 3 != 0:
+                if self.number_of_disk / 3 == 1:
+                    self.disk = self.disk[:-1]
+                else:
+                    self.disk = self.disk[:-2]
+
+        if self.value[2] == 'Even':
+            if self.number_of_disk % 2 != 0:
+                self.disk = self.disk[:-1]
+        self.raid_disk = ",".join(self.disk).strip(" ")
+        if self.raid_level == 'r10' or self.raid_level == 'r00':
+            self.pdperarray = 2
+        elif self.raid_level == 'r50' or self.raid_level == 'r60':
+            self.pdperarray = 3
+
+    def test_createall(self):
+
+        """
+        Function to create different raid level
+        """
+        write_policy = ['WT', 'WB', 'AWB']
+        read_policy = ['nora', 'ra']
+        io_policy = ['direct', 'cached']
+        strip = [64, 128, 256, 512, 1024]
+        for i in write_policy:
+            for j in read_policy:
+                for k in io_policy:
+                    for stripe in strip:
+                        self.vd_create(i, j, k, stripe)
+                        self.vd_details()
+                        self.vd_delete()
+
+    def test_maxvd(self):
+
+        """
+        Function to create max VD
+        """
+        for i in range(1, 17):
+            self.vd_create('WT', 'nora', 'direct', 1024)
+        self.vd_details()
+        self.vd_delete()
+        if self.raid_level == 'r0':
+            for i in range(0, 3):
+                if i > self.number_of_disk:
+                    self.raid_disk = self.disk[i]
+                    for i in range(1, 17):
+                        self.vd_create('WT', 'nora', 'direct', 512)
+                    self.vd_details()
+                self.vd_delete()
+
+    def test_cc(self):
+
+        """
+        Function to do consistency operations on VD
+        """
+        if self.raid_level == 'r0':
+            self.fail("CC is not applicable for Raid0")
+        self.vd_create('WT', 'nora', 'direct', 256)
+        self.vd_details()
+        cmd = "./storcli64 /c%d/v0 start cc force" % self.controller
+        self.check_pass(cmd, "Failed to start CC")
+        time.sleep(10)
+        self.cc_showprogress()
+        cmd = "./storcli64 /c%d/v0 stop cc" % self.controller
+        self.check_pass(cmd, "Failed to stop CC")
+        time.sleep(10)
+        self.cc_showprogress()
+        cmd = "./storcli64 /c%d/v0 start cc force" % self.controller
+        self.check_pass(cmd, "Failed to start CC")
+        time.sleep(10)
```

Comment:
If I move this sleep function in check_pass function, wont it wait ten second for everytime I try to run the process command?